config/electron.config.js

```diff
@@ -1,23 +1,25 @@
 const ENVIRONMENT = require('../src/CONST/ENVIRONMENT');
+const {version} = require('../package.json');
 
 module.exports = {
     appId: 'com.expensifyreactnative.chat',
     productName: 'New Expensify',
     extraMetadata: {
-        main: './desktop/main.js',
+        version,
         electronEnvironment: process.env.SHOULD_DEPLOY_PRODUCTION ? ENVIRONMENT.PRODUCTION : ENVIRONMENT.STAGING,
     },
     mac: {
         category: 'public.app-category.finance',
+        target: [
+            {target: 'dmg', arch: ['x64', 'arm64', 'universal']},
+        ],
         icon: process.env.SHOULD_DEPLOY_PRODUCTION === 'true' ? './desktop/icon.png' : './desktop/icon-stg.png',
         hardenedRuntime: true,
         entitlements: 'desktop/entitlements.mac.plist',
         entitlementsInherit: 'desktop/entitlements.mac.plist',
         type: 'distribution',
     },
     dmg: {
-        title: 'New Expensify',
-        artifactName: 'NewExpensify.dmg',
```

Comment:
Removing these adds the version to the file name e.g. `New Expensify-1.1.38-3.dmg` the title is preserved 

![image](https://user-images.githubusercontent.com/12156624/153924394-c873f047-2b9b-424d-9173-56853a16d52b.png)